roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -8,6 +8,12 @@ assetConfig:
   logoutURL: ""
   masterPublicURL: {{ openshift.master.public_api_url }}
   publicURL: {{ openshift.master.public_console_url }}/
+{% if openshift.master.logging_public_url %}
```

Comment:
Similar to @abutcher's comment in the previous PR, the conditional should be `{% if 'logging_public_url' in openshift.master %}`

Automated test failure was:
>TASK: [openshift_master | Create master config] *******************************
>fatal: [10.8.52.78] => {'msg': "AnsibleUndefinedVariable: One or more undefined variables: 'dict object' has no attribute 'logging_public_url'", 'failed': True}
>fatal: [10.8.52.78] => {'msg': "AnsibleUndefinedVariable: One or more undefined variables: 'dict object' has no attribute 'logging_public_url'",